notes/whats-new-csaf-v2.0-cn01.md

```diff
@@ -377,6 +377,7 @@ This informative appendix provides a mapping by path between the elements in CSA
 * `/product_tree/*/branches[]/category`: The values `Realm` and `Resource` have been removed. See conversion rule in [section 9.1.5 of CSAF specification](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#915-conformance-clause-5-cvrf-csaf-converter).
 * `/product_tree/relationships[]/full_product_name`: See conversion rule in [section 9.1.5 of CSAF specification](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#915-conformance-clause-5-cvrf-csaf-converter).
 * `/vulnerabilities[]/ids`: The IDs element was changed into an array to be able to contain multiple IDs. See conversion rule in [section 9.1.5 of CSAF specification](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#915-conformance-clause-5-cvrf-csaf-converter).
+* `/vulnerabilities[]/remediations[]`: See conversion rules in [section 9.1.5 of CSAF specification](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#915-conformance-clause-5-cvrf-csaf-converter). Remediations are explicitly now bound to Product or Group IDs.
```

Comment:
Maybe reconsider the `now` (placement and existence of the word) within:
```
Remediations are explicitly now bound to Product or Group IDs
```

Reply:
Removed in [a443efa](https://github.com/oasis-tcs/csaf/pull/496/commits/a443efa44fd59bfade8b68846d7c76de88e572de).